### ISSUE TITLE
Invincible To Some Players

### ISSUE THREAD

**Issue by FuzzNuggettt** [MinigameMC/Issues#56]
During PVP, from time to time you run into some players who you cannot hit/kill, yet other people can. Once you die, you can hit/kill them. Seems to be quite random, I haven't noticed a pattern as to why/when it happens.

**Comment by JayNZ** [comment]
Does it seem as if they are in creative mode?

**Comment by FuzzNuggettt** [comment]
Yeah well it's as if they have god mode. They don't have any creative abilities, you just can't hit them.
